src/v2/wallet.js

```diff
@@ -1116,9 +1114,27 @@ Wallet.prototype.prebuildAndSignTransaction = function(params, callback) {
       throw error;
     }
 
+    // call prebuildTransaction and keychains-get in parallel
     // the prebuild can be overridden by providing an explicit tx
-    const txPrebuild = params.prebuildTx || (yield this.prebuildTransaction(params));
-    const userKeychain = yield this.baseCoin.keychains().get({ id: this._wallet.keys[0], reqId: params.reqId });
+    const queries = params.prebuildTx ? [Promise.resolve(params.prebuildTx)] : [this.prebuildTransaction(params)];
+    queries.push(this.baseCoin.keychains().get({ id: this._wallet.keys[0], reqId: params.reqId }));
+    queries.push(this.baseCoin.keychains().get({ id: this._wallet.keys[1], reqId: params.reqId }));
```

Comment:
we actually only want to query for backup and bitgo keys if they are not explicitly provided in the verificationParams.